docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Mattermost/Nagios Plugin
+
+A Nagios plugin for Mattermost. Supports Nagios Core >= 4.4.x.
+
+This plugin allows you to:
+
+* Get logs from specific systems without leaving Mattermost.
+* Get alerts and notifications resembling the `showlog.cgi` UI instantly delivered.
+* Receive system monitoring reports on a subscribed channel.
+* Be frequently informed which hosts and services have an abnormal state.
+* Receive notifications about changes to the configuration on a subscribed channel.
+* Receive a diff between the old and the new version anytime a change has been made to Nagios configuration.
+
+### Getting logs
```

Comment:
It's a third-level header so that it doesn't appear in the TOC as there's no content, only screenshots. 

I guess it would also work as simple bolded headers. :)